### PR TITLE
add translations for ko locale

### PR DESCRIPTION
**Pull request by cs09g**

### CODE REVIEW

**Comment by SphinxKnight** [comment]
@alattalatta if you do have the time (and if I'm not mistaken on the fact you are a Korean native speaker as well), I would appreciate your check regarding the language :) 

Regarding the structure of the JSON it looks pretty ok to me :)

**Comment by cs09g** [comment]
FYI, I'm a Korean reviewer on SUMO(https://support.mozilla.org/ko/user/seulgi). :grinning:

**Comment by SphinxKnight** [comment]
OK OK ^^ my real bad :/ let's mark this as OK then 😄 

@escattone or anyone from the team, if you are ok technically-speaking, let's merge this one

**Comment by cs09g** [comment]
I'm not perfect though, any suggestions are welcome :)

File: macros/AddonSidebar.ejs [hunk 1]
```diff
@@ -273,6 +273,97 @@ var text = mdn.localStringMap({
...
+    'WebExtensions/Implement_a_settings_page': '설정 페이지 구현하기',
+    'WebExtensions/Working_with_the_Tabs_API': 'Tabs API 사용하기',
+    'WebExtensions/Work_with_the_Bookmarks_API': 'Bookmarks API 사용하기',
+    'WebExtensions/Work_with_the_Cookies_API': 'Cookies API 사용하기',
+    'WebExtensions/Work_with_contextual_identities': 'contextual identities 사용하기',
+    'WebExtensions/onboarding_upboarding_offboarding_best_practices': '사용자의 온보드, 업보드 및 오프보드',
+    'WebExtensions#Porting': '포팅',
+    'WebExtensions/Firefox_differentiators': 'Firefox 의 차별화',
+    'WebExtensions/Porting_a_Google_Chrome_extension': 'Google Chrome 확장 기능 포팅',
+    'WebExtensions/Porting_a_legacy_Firefox_add-on': 'P레거시 Firefox 확장 기능 포팅',
```
Comment (alattalatta) [inline macros/AddonSidebar.ejs]:
```suggestion
    'WebExtensions/Porting_a_legacy_Firefox_add-on': '레거시 Firefox 확장 기능 포팅',
```
A typo?

File: macros/AddonSidebar.ejs [hunk 2]
```diff
@@ -273,6 +273,97 @@ var text = mdn.localStringMap({
...
+    'WebExtensions/Safely_inserting_external_content_into_a_page': '외부 컨텐츠 삽입하기',
+    'WebExtensions/Add_a_button_to_the_toolbar': '도구 모음에 버튼 추가하기',
+    'WebExtensions/Implement_a_settings_page': '설정 페이지 구현하기',
+    'WebExtensions/Working_with_the_Tabs_API': 'Tabs API 사용하기',
+    'WebExtensions/Work_with_the_Bookmarks_API': 'Bookmarks API 사용하기',
+    'WebExtensions/Work_with_the_Cookies_API': 'Cookies API 사용하기',
+    'WebExtensions/Work_with_contextual_identities': 'contextual identities 사용하기',
+    'WebExtensions/onboarding_upboarding_offboarding_best_practices': '사용자의 온보드, 업보드 및 오프보드',
+    'WebExtensions#Porting': '포팅',
+    'WebExtensions/Firefox_differentiators': 'Firefox 의 차별화',
```
Comment (alattalatta) [inline macros/AddonSidebar.ejs]:
```suggestion
    'WebExtensions/Firefox_differentiators': 'Firefox의 차별화',
```
It's my personal opinion, but I prefer not to add a space.

Same for :322, :330

**Comment by alattalatta** [comment]
Other than one typo it looks nice. I don't have a strong opinion for spaces between English words and Korean postpositions but it's just that I'm not inserting any at documents of MDN.

**Comment by cs09g** [comment]
@alattalatta Thanks for the review! For the space, it's fair enough but I prefer having space after english keyword like brand name. Sometimes, or for some font, its look seems weird when they're sticked.

**Pull request merged**